docs/release/releasing.md

```diff
@@ -1,16 +1,20 @@
 # Releasing
 
-## Gradle
+## Releasing new version od db-replica
```

Comment:
How about this?
Give mefeedback if you have a bigger idea how it should look like.
I 'm open to edit it.